core/src/main/scala/org/apache/spark/BarrierCoordinator.scala

```diff
@@ -65,7 +65,7 @@ private[spark] class BarrierCoordinator(
 
   // Record all active stage attempts that make barrier() call(s), and the corresponding internal
   // state.
-  private val states = new ConcurrentHashMap[ContextBarrierId, ContextBarrierState]
+  private[spark] val states = new ConcurrentHashMap[ContextBarrierId, ContextBarrierState]
```

Comment:
Could you turn the `// ...` comment into ScalaDoc `/** ... */` and mention `Visible for unit testing.` in the doc?

Reply:
No problem, done in ecf12bd.